libs/entitlement/src/test/java/org/elasticsearch/entitlement/runtime/policy/PolicyParserTests.java

```diff
@@ -52,6 +52,35 @@ public void testPolicyBuilderOnExternalPlugin() throws IOException {
         assertEquals(expected, parsedPolicy);
     }
 
+    public void testParseFile() throws IOException {
+        Policy parsedPolicy = new PolicyParser(new ByteArrayInputStream("""
+            entitlement-module-name:
+              - file:
+                  path: test/path/to/file
```

Comment:
This path should be rejected, right? We're allowing only absolute paths, so they should always start with `/`.

Reply:
It's unrelated; this is copying the existing test we have in the resources as we were missing an explicit file parsing test. I'll remove it, and we'll fix this when the time comes.